File: avril2025/kit_démarage_python/bot.py
import random
from time import time

class Bot:
    NOM : str = "Super Bot Python!" + str(time())
    SERVEUR_REP : str = "../serveur_src/serveur.py" # Répertoire vers l'exécutable du serveur. Si non fournit, présumerra qu'il est déjà en marche
    ADRESSE : str = "127.0.0.1:5000" # Adresse pour se connecter au serveur

    def __init__(self):
        pass

    def tour(self, dernier_tour : dict[str:any]) -> bool:
        print(dernier_tour)
        # Vous pouvez faire mieux ;)
        return random.choice([True,False])

File: avril2026/kit_démarage_java/src/Bot.java
public class Bot {
    public static final String NOM = "SuperJavaBot"+Long.toString(System.currentTimeMillis());
    public static final String SERVEUR_REP = null;
    public static final String ADRESSE = "http://127.0.0.1:5000";

    public Bot(){}

    public boolean tour(ÉtatJeu jeu){
        System.out.println(jeu);
        // Vous pouvez faire mieux ;)
        return Math.random() > 0.5;
    }
}
